### PR TITLE
Bump data package

### PR DESCRIPTION
**Pull request by bobbyu99**
Bump data package after api release

### CODE REVIEW

File: scripts/cloud-e2e.sh
Comment (bobbyu99) [inline scripts/cloud-e2e.sh:1]: Didn't realize I merged the changes to this branch instead of dev

File: scripts/update-data-dependencies.sh
Comment (bobbyu99) [inline scripts/update-data-dependencies.sh:1]: bump minor versions only

File: scripts/cloud-e2e.sh [hunk 1]
```diff
@@ -16,10 +16,13 @@ function cloudE2EBeta {
 }
 function cloudE2E {
     echo Running Prod E2E Test Suite
-    if [[-z "$USER"]]; then
+    if [[ -n "$USER" ]]; then
```
Comment (bobbyu99) [inline scripts/cloud-e2e.sh:19]: Didn't realize I merged the changes to this branch instead of dev

File: scripts/update-data-dependencies.sh [hunk 2]
```diff
@@ -13,6 +13,7 @@ if [ $# -eq 0 ]
         --deep \
         --upgrade \
         --dep "prod,dev,peer,bundle,optional" \
+        --target minor \
```
Comment (bobbyu99) [inline scripts/update-data-dependencies.sh:16]: bump minor versions only

File: .circleci/local_publish_helpers_codebuild.sh [hunk 3]
```diff
@@ -139,10 +139,10 @@ function verifyPkgCli {
...
     }
 
-    verifySinglePkg "amplify-pkg-linux-x64" "amplify-pkg-linux-x64.tgz" $((830 * 1024 * 1024))
-    verifySinglePkg "amplify-pkg-macos-x64" "amplify-pkg-macos-x64.tgz" $((835 * 1024 * 1024))
-    verifySinglePkg "amplify-pkg-win-x64.exe" "amplify-pkg-win-x64.tgz" $((820 * 1024 * 1024))
-    verifySinglePkg "amplify-pkg-linux-arm64" "amplify-pkg-linux-arm64.tgz" $((670 * 1024 * 1024))
+    verifySinglePkg "amplify-pkg-linux-x64" "amplify-pkg-linux-x64.tgz" $((865 * 1024 * 1024))
+    verifySinglePkg "amplify-pkg-macos-x64" "amplify-pkg-macos-x64.tgz" $((870 * 1024 * 1024))
+    verifySinglePkg "amplify-pkg-win-x64.exe" "amplify-pkg-win-x64.tgz" $((860 * 1024 * 1024))
+    verifySinglePkg "amplify-pkg-linux-arm64" "amplify-pkg-linux-arm64.tgz" $((710 * 1024 * 1024))
```
Comment (sobolk) [inline .circleci/local_publish_helpers_codebuild.sh:145]: Do we know what contributed to this change?
Could this be avoided ?

File: yarn.lock [hunk 4]
```diff
@@ -1451,20 +1471,20 @@ __metadata:
...
     "@aws-amplify/graphql-transformer-core": 2.9.3
     "@aws-amplify/graphql-transformer-interfaces": 3.10.1
-    "@aws-sdk/client-ec2": 3.338.0
-    "@aws-sdk/client-iam": 3.338.0
-    "@aws-sdk/client-lambda": 3.338.0
-    "@aws-sdk/client-rds": 3.338.0
+    "@aws-sdk/client-ec2": 3.624.0
+    "@aws-sdk/client-iam": 3.624.0
+    "@aws-sdk/client-lambda": 3.624.0
+    "@aws-sdk/client-rds": 3.624.0
```
Comment (sobolk) [inline yarn.lock:1487]: By looking at yarn.lock file it seems that category api updated V3 CDK.
But on the CLI side we depend on older.
And it seems that some transitive dependencies in yarn.lock file got duplicated as a result. Which is most likely why binary size grew that much.
We should try to align SDK version on CLI side with what data brings and see if this resolves size problem.

Comment (bobbyu99) [reply yarn.lock]: Should I try to align all SDK version, not just the 4 deps you quoted above?

Comment (sobolk) [reply yarn.lock]: yes. they all share common transitive dependencies. it looks like those got duplicated due to mismatches.

Comment (sobolk) [reply yarn.lock]: All v3 SDKs to be precise.

**Review by sobolk — APPROVED** [review]

**Review by palpatim — APPROVED** [review]

**Pull request merged**